project/Publish.scala

```diff
@@ -69,17 +69,18 @@ object DeployRsync extends AutoPlugin {
   override def requires = plugins.JvmPlugin
 
   trait Keys {
-    val deployRsyncArtifact = taskKey[Seq[(File, String)]]("File or directory and a path to deploy to")
-    val deployRsync = inputKey[Unit]("Deploy using SCP")
+    val deployRsyncArtifacts = taskKey[Seq[(File, String)]]("File or directory and a path to deploy to")
+    val deployRsync = inputKey[Unit]("Deploy using rsync")
   }
 
   object autoImport extends Keys
   import autoImport._
 
-  override def projectSettings =
-    Seq(deployRsync := {
+  override def projectSettings = Seq(
+    deployRsyncArtifacts := List(),
+    deployRsync := {
       val (_, host) = (Space ~ StringBasic).parsed
-      deployRsyncArtifact.value.foreach {
+      deployRsyncArtifacts.value.foreach {
         case (from, to) => s"rsync -rvz $from/ $host:$to" !
```

Comment:
this is not using the same gustav credentials as the snapshot publishing? is there a value in using the same?

Reply:
It would be nice if they could use the same. I would like it if the sbt publishing infrastructure would be able to pick up the system keys that are also used by rsync. I would also like to reuse more of that infrastructure to replace our own DeployRsync with that. Right now this is easy and works, though ;).